### PR TITLE
chore(functions): add node termination example

### PR DESCRIPTION
**Pull request by ace-n**
Related to b/176872447

### CODE REVIEW

File: functions/concepts/index.js [hunk 1]
```diff
@@ -104,3 +104,18 @@ exports.makeRequest = async (req, res) => {
   res.sendStatus(externalRes.ok ? 200 : 500);
 };
 // [END functions_concepts_requests]
+
+exports.nodeTermination = async (event, context, callback) => {
+  // [START functions_concepts_node_termination]
```
Comment (grant) [inline functions/concepts/index.js:109]: What region tag is this? Is this a new sample?

**Review by grant — CHANGES_REQUESTED** [review]
Please add a PR description.

File: functions/concepts/index.js (hunk 1 above)
Comment (ace-n) [reply functions/concepts/index.js:109]: Yes - see b/176872447

File: functions/concepts/index.js [hunk 2]
```diff
@@ -104,3 +104,18 @@ exports.makeRequest = async (req, res) => {
   res.sendStatus(externalRes.ok ? 200 : 500);
 };
 // [END functions_concepts_requests]
+
+exports.nodeTermination = async (event, context, callback) => {
```
[the remark this answers is not in the document]
Comment (ace-n) [reply functions/concepts/index.js:108]: We call the `callback` function, so yes - we do need it.

File: functions/concepts/termination.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/* eslint-disable */
+
+exports.nodeTermination = async (event, context, callback) => {
```
Comment (grant) [inline functions/concepts/termination.js:19]: We cannot have both `async` and `callback` in the same function. The Functions Framework won't resolve this correctly.

Either write:

```js
exports.nodeTermination = async (event, context) => {
```

or

```js
exports.nodeTermination = (event, context, callback) => {
```

**Review by grant — CHANGES_REQUESTED** [review]

File: functions/concepts/termination.js (hunk 3 above)
Comment (ace-n) [reply functions/concepts/termination.js:19]: To be clear, this function is not actually meant to be run. (It's purely there to serve as a "wrapper" for the lines inside.)

(I can add a comment to that effect, if you want.)

Comment (grant) [reply functions/concepts/termination.js:19]: Just remove `callback` or remove `async`. If we need both, we need 2 style of functions.

Comment (ace-n) [reply functions/concepts/termination.js]: Done.

**Review by grant — APPROVED** [review]

**Pull request merged**